### PR TITLE
add blog post: Using Springdoc-openapi with resteasy-spring-boot

### PR DESCRIPTION
**Pull request by liweinan**

### CODE REVIEW

File: _posts/2024-11-15-resteasy-spring-boot-with-springdoc.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+author:     Wei Nan Li
+---
+
+Recently there is a request in the `resteasy-spring-boot` community:
+
+- [Can not integrate with spring doc · Issue \#349 · resteasy/resteasy-spring-boot](https://github.com/resteasy/resteasy-spring-boot/issues/349)
+
+It’s an interesting request because I haven’t dug into this kind of usage before. I spent some time investigating the topic, and I’d like to share what I have learned in this blog post.
+
+The OpenAPI is a specification[^openapi_spec] by itself, and it doesn’t enforce its implementations. There are currently two major implementors of the specification. One is the [MicroProfile OpenAPI Specification](https://download.eclipse.org/microprofile/microprofile-open-api-1.1.2/microprofile-openapi-spec.html) and the other one is the [Swagger OpenAPI Specification - Version 3.1.0](https://swagger.io/specification/). These two specifications define their own Java annotations, service provider interfaces, and configurations. The [smallrye-open-api](https://github.com/smallrye/smallrye-open-api) project implements the MicroProfile OpenAPI Specification, on the other side, the Swagger project implements its specification.
```
Comment (jamezp) [inline _posts/2024-11-15-resteasy-spring-boot-with-springdoc.md:15]: I"m not sure if I have a better idea, but I don't know that I'd say the MicroProfile OpenAPI is an implementation. It's more of an API for OpenAPI and Java of which SmallRye OpenAPI is the implementation. I'm not sure how to word that though :)

**Review by jamezp — APPROVED** [review]
This looks good to me. I left one comment, but I'm not really sure what is the correct wording there. If you think it seems fine, feel free to merge it

File: _posts/2024-11-15-resteasy-spring-boot-with-springdoc.md (hunk 1 above)
Comment (liweinan) [reply _posts/2024-11-15-resteasy-spring-boot-with-springdoc.md]: Thanks for checking James! I have added more explanations to this part.

Comment (liweinan) [reply _posts/2024-11-15-resteasy-spring-boot-with-springdoc.md]: If you feel the addition/change of the text it okay I'll merge this.

**Review by jamezp — APPROVED** [review]

**Comment by liweinan** [comment]
Relative reference link fixed: https://github.com/pe-st/apidocs/pull/8

**Pull request merged**